tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,312 @@
+  }
+  fmt.Println(dbName)
+}
+```
+
+Parameter description：
+
+- Register `tls.Config` in connection to enable TLS and validate TiDB Dedicated Tier clusters. 
+- Set `MinVersion: tls.VersionTLS12` to restrict the versions of TLS protocol.
+- Set `ServerName: "<host>"` to verify TiDB Dedicated Tier's hostname.
+- If you do not want to register a new TLS configuration, you can just set `tls=true` in the connection string.
+
+</div>
+
+<div label="Node.js">
+
+[Mysql](https://www.npmjs.com/package/mysql2)'s TLS connection configurations are used here as an example.
+
+```
+var connection = mysql.createConnection({
+  host: 'tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com',
+  port: 4000,
+  user: 'root',
+  password: '<your_password>',
+  database: 'test',
+  ssl: {
+    ca: fs.readFileSync('ca.pem'),
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true
+  }
+});
+```
+
+You can click **show example usage** to view detailed code examples.
+
+```
+var mysql = require('mysql2');
+var fs = require('fs');
+var connection = mysql.createConnection({
+  host: 'tidb.srgnqxji5bc.clusters.staging.tidb-cloud.com',
+  port: 4000,
+  user: 'root',
+  password: '<your_password>',
+  database: 'test',
+  ssl: {
+    ca: fs.readFileSync('ca.pem'),
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true
+  }
+});
+connection.connect(function(err) {
+  if (err) {
+    throw err
+  }
+  connection.query('SELECT DATABASE();', function(err, rows) {
+    if (err) {
+      throw err
+    }
+    console.log(rows[0]['DATABASE()']);
+    connection.end()
+  });
+});
+```
+
+Parameter description：
+
+- Set `ssl: {minVersion: 'TLSv1.2'}` to restrict the versions of TLS protocol.
+- Set `ssl: {ca: fs.readFileSync('ca.pem')}` to read the special path ca.pem.
+
+
+</div>
+</SimpleTab>
+
+## Create and Reset Dedicated database password
+
+When you connect to your cluster for the first time, you should click **Security Setting** to automatically generate a root password with a length of 16 characters, including numbers, uppercase and lowercase characters, and special characters. 
+
+## Root Digital certificate management of Dedicated Tier 
+
+### Root Digital certificate‘s issuance and validity  
+
+TiDB Dedicated Tier uses certificates from [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Dedicated Tier clusters. Usually, the root certificate of ACM are stored securely in AWS managed hardware security modules (HSMs) that adhere to [FIPS 140-2 Level 3](https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3139) security standards. 
+
+> **Note:**
+>
+After you download your Dedicated Tier cluster CA to the local operating system, you can store it in the system default storage directory, or customize the TiDB Cluster CA storage directory. It should be noted that you need to add your local real TiDB Cluster CA path to the code example.
+
```

Comment:
```suggestion
```

Reply:
reason: move this note to step 5 under  "Secure connection to a Dedicated Tier cluster" because users need to pay attention to that in step 6